av/utils.pyx

```diff
@@ -84,9 +84,3 @@ cdef flag_in_bitfield(uint64_t bitfield, uint64_t flag):
     if not flag:
         return None
     return bool(bitfield & flag)
-
-
-# === BACKWARDS COMPAT ===
-
-from .error import FFmpegError as AVError
-from .error import err_check
```

Comment:
Removing these is technically a breaking change, so we can put them back if we're concerned about that